### ISSUE TITLE
Error while training the model

### ISSUE THREAD

**Issue by mateusjunges** [marcogdepinto/Emotion-Classification-Ravdess#10]
Good night! 
I'm trying out your emotion classification system, but i couldn't train the model. Can you help me?

Here is a screenshot from my jupyter notebook?
![image](https://user-images.githubusercontent.com/19756164/80894362-c4140300-8cb0-11ea-81bc-1f7b86c8fa45.png)

**Comment by abdoulsn** [comment]
What is activation_11 is that the flattened layer of input? Pleas provide input data shape and model.summary(). May I can help you.

**Comment by mateusjunges** [comment]
This is my model summary?
![image](https://user-images.githubusercontent.com/19756164/80894458-67651800-8cb1-11ea-8558-a332db09e0ae.png)


And if i print `x_train_cnn.shape` and `x_test_cnn.shape` i have this output:

```
x_train_cnn.shape, x_test_cnn.shape
((724, 40, 1), (311, 40, 1))
```

**Comment by abdoulsn** [comment]
Thanks, It a problem on output.

If your Y is One-Hot_Encoded (OHE), use **categorical_crossentropy**.
Examples of OHE shape:
    [1,0,0]
    [0,1,0] 
    [0,0,1]

But if your targets are label encoded, use **sparse_categorical_crossentropy**.
Examples of integer encodings (for the sake of completion):
    1
    2
    3
So try to play with your **loss function**. Let me know.

**Comment by mateusjunges** [comment]
@abdoulsn I changed the sparse_categorical_crossentropy to categorical_crossentropy, but i still got the same issue.

My model definition is the same, but the name of the layers are differs from the above one because i need to restart the jupyter notebook.

This is how i am compiling the model:

```python

model.compile(loss='categorical_crossentropy',
              optimizer=opt,
              metrics=['accuracy'])
```

Here's my model summary:
![image](https://user-images.githubusercontent.com/19756164/80895355-5e2c7900-8cba-11ea-8d13-0f9ba1e64251.png)

And this is the error:

![image](https://user-images.githubusercontent.com/19756164/80895368-7bf9de00-8cba-11ea-98b3-ce32f97392b2.png)

**Comment by abdoulsn** [comment]
It's sure in Y train shape, What is your Y shape? one column or multiple binary column? 
Pleas check this also https://github.com/keras-team/keras/issues/3109#issuecomment-229634628

**Comment by mateusjunges** [comment]
@abdoulsn Thanks for your help, i fixed my code.
